### ISSUE TITLE
make mini map interactive?

### ISSUE THREAD

**Issue by jiyoungyoun** [knightjdr/prohits-viz-client#13]
As we rearrange large dotplots, it would be nice to have the mini map view to be updated to the newly sorted/manually arranged view. Would this be possible?

**Comment by knightjdr** [comment]
Can you clarify what you're doing?

If you change the main image (sorting) the minimap will display a sync button you can use to update. This isn't done automatically in case someone is doing a lot of sorting and doesn't care about the minimap it just wastes time.

There is no minimap when you are on the customize view. My assumption here was that most people would have relatively small images for customization so a minimap wouldn't be necessary but I have a feeling that may have been an incorrect assumption.

**Comment by jiyoungyoun** [comment]
Yes, I see that the sync function works fine when you're sorting in the main image. 
But when working with the customized view, the mini map does not sync. I often have large dotplots after customizing my baits (* I often do not remove preys to still keep the global list). It would be nice to be able to sync the mini-map in customized view as well.
